plugins/woocommerce/tests/api-core-tests/tests/customers/customers-crud.test.js

```diff
@@ -0,0 +1,451 @@
+
+		test(`can update a customer`, async ({
+			request,
+		}) => {
+			// update customer names (regular, billing and shipping) from John to Jack
+			const response = await request.put(
+				`/wp-json/wc/v3/customers/${ customerId }`, {
+					data: {
+						first_name: 'Jack',
+						billing: {
+							first_name: 'Jack'
+						},
+						shipping: {
+							first_name: 'Jack'
+						}
+					}
+				}
+			);
+			const responseJSON = await response.json();
+			expect(response.status()).toEqual(200);
+			expect(responseJSON.first_name).toEqual('Jack');
+			expect(responseJSON.billing.first_name).toEqual('Jack');
+			expect(responseJSON.shipping.first_name).toEqual('Jack');
+		});
+
+		test('retrieve after update customer', async ({
+			request
+		}) => {
+			// call API to retrieve all customers
+			const response = await request.get(`/wp-json/wc/v3/customers/${ customerId }`);
+			const responseJSON = await response.json();
+			expect(response.status()).toEqual(200);
+			expect(responseJSON.first_name).toEqual('Jack');
+			expect(responseJSON.billing.first_name).toEqual('Jack');
+			expect(responseJSON.shipping.first_name).toEqual('Jack');
+		});
+	});
+
+	test.describe('Delete a customer', () => {
+		test('can permanently delete an customer', async ({
+			request
+		}) => {
+			// Delete the customer.
+			const response = await request.delete(
+				`/wp-json/wc/v3/customers/${ customerId }`, {
+					data: {
+						force: true,
+					},
+				}
+			);
+			expect(response.status()).toEqual(200);
+
+			// Verify that the customer can no longer be retrieved.
+			const getDeletedCustomerResponse = await request.get(
+				`/wp-json/wc/v3/customers/${ customer }`
+			);
+			expect(getDeletedCustomerResponse.status()).toEqual(404);
+		});
+	});
+
+
+	test.describe('Batch update customers', () => {
+		/**
+		 * 2 Customers to be created in one batch.
+		 */
+		const expectedCustomers = [
+            {
+				email: "john.doe2@example.com",
+				first_name: "John",
+				last_name: "Doe",
+				username: "john.doe2",
+				billing: {
+					first_name: "John",
+					last_name: "Doe",
+					company: "",
+					address_1: "969 Market",
+					address_2: "",
+					city: "San Francisco",
+					state: "CA",
+					postcode: "94103",
+					country: "US",
+					email: "john.doe@example.com",
+					phone: "(555) 555-5555"
+				},
+				shipping: {
+					first_name: "John",
+					last_name: "Doe",
+					company: "",
+					address_1: "969 Market",
+					address_2: "",
+					city: "San Francisco",
+					state: "CA",
+					postcode: "94103",
+					country: "US"
+				}
+			},
+			{
+				email: "joao.silva2@example.com",
+				first_name: "João",
+				last_name: "Silva",
+				username: "joao.silva2",
+				billing: {
+					first_name: "João",
+					last_name: "Silva",
+					company: "",
+					address_1: "Av. Brasil, 432",
+					address_2: "",
+					city: "Rio de Janeiro",
+					state: "RJ",
+					postcode: "12345-000",
+					country: "BR",
+					email: "joao.silva@example.com",
```

Comment:
We should match the email up:

```suggestion
					email: "joao.silva2@example.com",
```